picard/util/thread.py

```diff
@@ -36,12 +36,15 @@ def run(self):
 
 class Runnable(QRunnable):
 
-    def __init__(self, func, next):
+    def __init__(self, func, next, os_priority=0):
```

Comment:
I accept that this is extra to the functionality of this PR. The idea is that it adjusts the **operating system priority** of the thread down (or up) in order to influence the way threads are dispatched by Python. But now I understand more about how the Python GIL works and impacts which thread gets the GIL (which is essentially random), I am not sure that it would be of any use. (And we don't know what else is running on the user's system and whether it would make sense to do it.)

So I will be happy to revert this Commit.